_data/talks.yml

```diff
@@ -56,23 +77,3 @@
   youtube-link: 'https://www.youtube.com/watch?v=vDQQ8axRC8I'
   featuredImage: mittie-talk
 
-- name: 'Java APIs - The missing manual'
```

Comment:
Is it the plan to move talk section to the beginning of the file instead of updating them in place?

Reply:
Currently the first on is on top of the rendered page. I think we should add a date or something to the data model and than sort it automatically ;) https://github.com/karakun/karakun.github.io/issues/97